src/shims/ApiCompatBaseline.netcoreapp.netstandard20Only.txt

```diff
@@ -1,4 +1,12 @@
 Compat issues with assembly netstandard:
 TypeCannotChangeClassification : Type 'System.RuntimeArgumentHandle' is a 'ref struct' in the implementation but is a 'struct' in the contract.
 TypeCannotChangeClassification : Type 'System.TypedReference' is a 'ref struct' in the implementation but is a 'struct' in the contract.
-Total Issues: 2
+CannotRemoveAttribute : Attribute 'System.ComponentModel.DefaultEventAttribute' exists on 'System.ComponentModel.BackgroundWorker' in the contract but not the implementation.
```

Comment:
Do we need to apply some of these to netstandard ref as well?

Reply:
These particular suppressions are needed because NETStandard had these attributes already.  These create layering issues in CoreFx so I didn't try to apply them.  We could, but we'd need to push attributes down.

For netstandard (I think you're suggesting the reverse) we could look at applying more attributes.  That's potentially breaking since not all implementations may have those attributes.  At least for the current issue it isn't a requirement since we aren't creating a netstandard-library-using-wpf/winforms scenario.